EIPS/eip-eth-vhash.md

```diff
@@ -0,0 +1,172 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 8094
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.